docker-unsupported/alpine-3.16-arm64-openssl-1.1.x-with-libc/README.md

```diff
@@ -0,0 +1,16 @@
+# OS support: fail-alpine-3.16-arm64-openssl-1.1.x-with-libc
+
+- Base Docker image: `node:lts-alpine3.16`
+- OS: Linux Alpine 3.16
+- Arch: arm64 (aarch64)
+- OpenSSL location: `/lib/libssl.so.1.1`
+- Binary target: `linux-arm64-openssl-1.1.x`
+
+Prisma doesn't support the `arm64` architecture for Linux Alpine.
```

Comment:
Do we have an issue that tracks that which we can link here? Then it will be easy to crossreference.

Reply:
https://github.com/prisma/prisma/issues/8478

Reply:
Added this to the `alpine` tests in `docker-unsupported`